File: Core/Entities/Booking.cs
using System;

namespace Core.Entities
{
    public class Booking: BaseEntity
    {
        public int CarId { get; set; }
        public string UserId { get; set; }
        public DateTime PickUpDate { get; set; }
        public DateTime DropOffDate { get; set; }
        public string PickUpLocation { get; set; }
        public string DropOffLocation { get; set; }
        public decimal TotalCost { get; set; }

        public virtual Car Car { get; set; }
        public virtual User User { get; set; }
    }
}

File: WebApp/ClientApp/src/types/Booking.js
class Booking {
	constructor(id = 0, carId = 0, userId = "", pickUpDate = "", dropOffDate = "", pickUpLocation = "", dropOffLocation) {
		this.id = id;
		this.carId = carId;
		this.userId = userId;
		this.pickUpDate = pickUpDate;
		this.dropOffDate = dropOffDate;
		this.pickUpLocation = pickUpLocation;
		this.dropOffLocation = dropOffLocation;
	}
}

export default Booking;
